### PR TITLE
Added test of rendering functions for report_delivery.py

### PR DESCRIPTION
**Pull request by keisetsu**
Fixes issue #19.

### CODE REVIEW

File: tests/callistocore/test_delivery.py [hunk 1]
```diff
@@ -19,6 +19,275 @@
...
+        }
+        ]'''
+        report = PDFFullReport(self.report, radio_question)
+        output = report.generate_pdf_report(recipient=None, report_id=None)
+        rendered_report = BytesIO(output)
+        pdf_reader = PyPDF2.PdfFileReader(rendered_report)
+
+        rendered_text = pdf_reader.getPage(1).extractText()
+        self.assertIn('A radiobutton question?', rendered_text)
+        for i in range(5):
```
Comment (kelseyq) [inline tests/callistocore/test_delivery.py]: any thoughts on verifying that the given answer is marked as checked here? open to changing the output to make checked vs unchecked a little more obvious. same question for checkbox questions

**Comment by kelseyq** [comment]
This is great! I had one question but if you think tackling it will be involved I'm fine with creating a spinoff bug just for that

File: tests/callistocore/test_delivery.py (hunk 1 above)
Comment (keisetsu) [inline tests/callistocore/test_delivery.py]: That bugged me, too; as far as I can tell the PDFFileReader is unable to make sense of the X symbol. There is a way to look at the raw PDF data, if I remember correctly, and there might be a way to put it together from other queues. I'll take another look at it, and if I just can't seem to get it to work, I'll try to come up with a reasonable mark that could work.

**Comment by keisetsu** [comment]
Yeah, that question is going to take a bit of research. If you want, I can
create an issue once you accept the PR.

**Comment by kelseyq** [comment]
I created #122 to track it, thanks!

**Pull request merged**